.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Tests
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+      - '*.rst'
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'docs/**'
+      - '*.rst'
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {name: Linux, python: '3.11', os: ubuntu-latest, tox: py311}
```

Comment:
Was it a conscious decision to have `3.11` as the latest version rather than `3.12`?